sopel/bot.py

```diff
@@ -1157,8 +1157,8 @@ def search_url_callbacks(self, url):
             The Python documentation for the `re.search`__ function and
             the `match object`__.
 
-        .. __: https://docs.python.org/3.6/library/re.html#re.search
-        .. __: https://docs.python.org/3.6/library/re.html#match-objects
+        .. __: https://docs.python.org/3.7/library/re.html#re.search
+        .. __: https://docs.python.org/3.7/library/re.html#match-objects
```

Comment:
Surely at least `re.search` could use an intersphinx reference instead of a ha(n|r)d-coded URL? It should be possible to <tt>:ref:\`match-objects\`</tt>, too (also in `sopel/plugins/rules.py`).

Perhaps should be changed in one of the doc cleanups you/we are planning, instead of this PR, but I'm still going to call attention to it because it's being touched here. 😁

Reply:
Probably. I'll leave that to a future PR, one that fix all the "Pattern" and "Match" not found errors. 😁

Reply:
Future PR then?

Reply:
Yeah, future is fine.